shell/browser/hid/electron_hid_delegate.cc

```diff
@@ -82,21 +85,28 @@ bool ElectronHidDelegate::HasDevicePermission(
     content::BrowserContext* browser_context,
     const url::Origin& origin,
     const device::mojom::HidDeviceInfo& device) {
-  return GetChooserContext(browser_context)
-      ->HasDevicePermission(origin, device);
+  auto* chooser_context = GetChooserContext(browser_context);
+  if (!chooser_context)
+    return false;
+  return chooser_context->HasDevicePermission(origin, device);
```

Comment:
We should more closely mirror upstream here, eg:
```suggestion
  return browser_context && GetChooserContext(browser_context->HasDevicePermission(origin, device);
```